README.md

```diff
@@ -329,12 +329,34 @@ env:
 
 ## HOW TO
 
-To print SHA1 hashe to a given revision you can use :
+### To print SHA1 hashe to a given revision you can use :
 
 ```ssh
 $ git rev-parse origin/master
 ```
 
+<br>
+
+### Import data from Google SpreadSheet with google-spreadsheets package :
+
+You have to connect to the Google Spreadsheet document and select :
+
+```
+Fichier > Publier pour le web
+```
+
+On displayed modal you have to select
+```
+> Tabs that you want to publish
+> Page web
+```
+Then click on **"Publier"** button
+
+Once the data are published you can execute your import script
+
+> **Don't forget to unpublish when you have done**
```

Comment:
since it's public data, not sure this is important to unpublish once it's done

Reply:
I saw with Clément and he doesn't ok to let publish these data